.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,25 @@
 <!--
 
-If this PR should trigger a release, make sure your title is prefixed with one of these:
+### Production Release
 
-- fix: (patch release)
-- feat: (minor release)
+To trigger a production release:
```

Comment:
Perhaps the use of the term 'trigger' here might give the impression that the release will go out immediately - more accurately it could be something along the lines of 'To add this PR to the next release'?

Reply:
Thanks for that. Just updated this